infra/scoped/parameters.tf

```diff
@@ -233,6 +233,23 @@ resource "aws_ssm_parameter" "auth0_reset_email_url_ttl" {
   )
 }
 
+resource "aws_ssm_parameter" "auth0_welcome_email_subject" {
+  name  = "identity-auth0_welcome_email_subject-${terraform.workspace}"
```

Comment:
Does this need to use the prefix you generated in the other PR?

Reply:
Nah, this isn't a user-facing resource. That qualifier is only for the resources that a user can actually see.